mmedit/datasets/sr_folder_ref_dataset.py

```diff
@@ -72,6 +73,7 @@ def __init__(self,
                  filename_tmp_gt='{}',
                  filename_tmp_lq='{}'):
         super().__init__(pipeline, scale, test_mode)
+        assert gt_folder or lq_folder
```

Comment:
Better add a message here (e.g. `At least one of 'gt_folder and 'lq_folder' cannot be None.`). Looks good to me other than this.